pkgs/tools/system/thinkfan/default.nix

```diff
@@ -1,15 +1,25 @@
-{ stdenv, fetchurl, cmake }:
+{ stdenv, fetchFromGitHub, cmakeCurses, libyamlcpp, libatasmart, pkgconfig }:
 
 stdenv.mkDerivation rec {
   name = "thinkfan-${version}";
-  version = "0.9.3";
+  version = "1.0.1";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/thinkfan/thinkfan-${version}.tar.gz";
-    sha256 = "0nz4c48f0i0dljpk5y33c188dnnwg8gz82s4grfl8l64jr4n675n";
+  src = fetchFromGitHub {
+    owner = "vmatare";
+    repo = "thinkfan";
+    rev = "${version}";
+    sha256 = "1983p8aryfgpyhflh5r5xz27y136a4vvm7plgrg44q4aicqbcp8j";
   };
 
-  nativeBuildInputs = [ cmake ];
+  configureFlags = [
+    "-DCMAKE_INSTALL_DOCDIR==share/doc/${name}"
```

Comment:
two equal signs?

Reply:
@ryantm I'm still learning and do not know how to do it right.

Reply:
One equal sign is enough.